library/webpack-loaders/css-loader.js

```diff
@@ -1,30 +1,34 @@
 const { relative, dirname } = require('path')
+const loaderUtils = require('loader-utils')
 const postcss = require('postcss')
 
 const plugins = [
   // these plugis need to run on each file individual file
   // look at the source of postcss-modules to see that it effectively runs all modules twice
-  ['postcss-plugin-composition', ({ onImport, onExport, resolve }) => [
+  ['postcss-plugin-composition', ({ onImport, onExport, resolve }, options) => [
```

Comment:
Instead of passing `options` I would probably pass another function in the arguments object, something like `getClassNamePattern`. Another option would be to rename `handlers` and put the options in there.